### PR TITLE
make PolylineArrowMaterial() actually work

### PR DESCRIPTION
**Pull request by mhaberler**
and fix the test while at it

### CODE REVIEW

File: src/czml3/properties.py [hunk 1]
```diff
@@ -41,7 +41,7 @@ class PolylineGlowMaterial(BaseCZMLObject):
 class PolylineArrowMaterial(BaseCZMLObject):
     """"A material that fills the surface of a line with an arrow."""
 
-    color = attr.ib(default=None)
+    polylineArrow = attr.ib(default=None)
```
Comment (astrojuanlu) [inline src/czml3/properties.py:44]: ...so the documentation is wrong? Or is there something I'm not following here? https://github.com/AnalyticalGraphicsInc/czml-writer/wiki/PolylineArrowMaterial

Comment (mhaberler) [reply src/czml3/properties.py:44]: yes, looks like the manual is wrong

I glanced the correct way of doing it from [this JSON fragment on github](https://github.com/lyqh-ctx/cesiumTx/blob/f6788936274f23c5b1d53671b25c1bcdc78db6e4/examples/czml_polyline.html#L81-L100)

Comment (astrojuanlu) [reply src/czml3/properties.py:44]: xref https://github.com/AnalyticalGraphicsInc/czml-writer/issues/181

Comment (astrojuanlu) [reply src/czml3/properties.py:44]: That example is good enough, thanks a lot!

**Review by astrojuanlu — APPROVED** [review]

**Comment by astrojuanlu** [comment]
Thanks @mhaberler !

**Comment by astrojuanlu** [comment]
Seems we got this one wrong: https://github.com/AnalyticalGraphicsInc/czml-writer/issues/181#issuecomment-745598944 reverting at #80.

**Pull request merged**